src/metabase/api/common.clj

```diff
@@ -424,9 +424,7 @@
   [limit offset]
   (do
     (check (not (and limit (not offset))) [400 (tru "When including a limit, an offset must also be included.")])
-    (check (not (and offset (not limit))) [400 (tru "When including an offset, a limit must also be included.")])
-    ))
-
+    (check (not (and offset (not limit))) [400 (tru "When including an offset, a limit must also be included.")])))
```

Comment:
Seems like an unneeded change?

Reply:
it looks like it's just a fix for hanging parens. Not _needed_, but I think it makes sense to just fix these things when we come across them rather than opening a whole separate PR to fix them. If we have to open a whole new PR to fix stuff like this it's just not going to get fixed

Reply:
I don't even remember changing this tbh. I think it was fixed automatically by my editor while I was snooping around in this file.